### DISCUSSION TITLE
Http trigger Link?

### DISCUSSION THREAD

**Discussion by SudgeFunday** [AntlerVC/firetable#1305] [Support / Q&A] [answered]
hello, 

so recently I created a derivative and I hoping I could get the HTTP trigger link out of it, so I headed back to Firebase and I do not see the link to http trigger there I just see "R-Orders". 
<img width="1251" alt="Screenshot 2023-06-19 at 8 10 38 AM" src="https://github.com/rowyio/rowy/assets/89611733/80620d04-abdf-400e-9b00-757d2b71678b">

It's not like the first Http trigger I created. 
<img width="1256" alt="Screenshot 2023-06-19 at 8 10 45 AM" src="https://github.com/rowyio/rowy/assets/89611733/84072423-0b98-40f4-9dd1-882e908d8618">

I'm wondering how I get the HTTP trigger from the derivative I created?

Thank You, 

samuel bundrant

**Comment by BeeBombshell** [comment, marked as answer]
Hey @SudgeFunday! To obtain the endpoint that triggers your functions, you can utilize Webhooks. Webhooks serve as a means to process incoming events from external services, enabling you to receive notifications about changes in your system. By employing the Webhook Parser, you can create a function that handles the incoming request, processes it, and optionally returns an object. This object can automatically add a new row to your database. Alternatively, you can write to any location in your database by utilizing the `db` parameter or `ref` to directly reference the collection in your table.

Webhooks Rowy Docs: https://docs.rowy.io/webhooks
